### ISSUE TITLE
Imagetest (Branch: 2018, Version 811x, bzw. a2bc98f2)

### ISSUE THREAD

**Issue by Nos-** [fsfw-dresden/usb-live-linux#124]
Legende: '[ ]' bedeutet noch zu testen, '[-]' Test nicht bestanden, '[x]' Test bestanden, ERGÄNZUNG steht für einen Ergänzungsvorschlag.

Hier mein Testergebnis des gestern Abend aus dem 2018-Branch gebauten Images (FSFW-Uni-Stick_811x__2018__KDE_stretch-amd64.hybrid.iso bzw. Commit: a2bc98f2):

* [-] ERGÄNZUNG: Loginpasswort (für Bildschirmsperre etc.)
	* [ ] sollte in der Begrüßungsdoku erwähnt sein und funktionieren
* Desktop (Hintergrund und Symbole)
	* [-] darauf achten, dass die Uni-Stick-spezifischen Programme räumlich von den allgemeinen Systemprogrammen getrennt sind.
* jupyter
	* [x] Konsolenfenster geht auf und Firefox öffnet einen neuen Tab
	* [x] Das Beispiel-Notebook lässt sich starten
	* [x] Zellen lassen sich ohne Fehlermeldung ausführen (SHIFT + Enter)
* zim
	* [x] Es ist eine Desktop-Verknüpfung dafür vorhanden
	* [x]  Beim Starten wird die FSFW-zim-Vorlage geöffnet
	* [x]  Über das Menü: Einfügen -> Gleichung bzw. ALT+E G erreicht man den Dialog zum einfügen von LaTeX-Formeln
	* [x]  folgender String sollte ein sinnvolles Ergebnis liefern: \alpha^2 + \beta = \frac{\pi}{2}
* Zotero
    * [-] Es sollte als Plugin im Webbrowser und LibreOffice installiert sein.
		* Firefoxplugin ist veraltet, Neuauflage (anderes deb-Paket) nötig
    * [ ] Beim ersten Start des Uni-Sticks sollte sich im Webbrowser die Zotero-Startseite nicht in den Vordergrund drängeln 
    * [ ] Webseiten oder manuell eingetragene Quellenangaben sollten sich über das Browseraddon erstellen und mittels des LibreOffice-Addon in Texte einfügen lassen
	* [ ] Zusätzlich sollten sich im Browser-Addon Quellen nach Bib(La)TeX (etwa für die Verwendung in LaTeX-Dokumenten) exportieren lassen
	* [ ] ERGÄNZUNG: Quellenangaben sollten auch in Libreoffice verwendbar sein
* Accessibilityverbesserung (Screenreader & Co.)
    * [ ] Brltty (Brailleterminal) soll beim booten automatisch und mit deutschsprachiger Konfig gestartet werden
    * [ ] Screenreader (Linux) und NVDA (Windows) sollen in der Doku (samt Tastenkombi zum Starten) erwähnt werden.
    * [ ] mit Gnome-orca/ espeak sollte ein Text vorgelesen werden können.
		* [-] Sound funzt in virtualisiertem System nicht und ich konnte es somit noch nicht testen
* LibreOffice
	* [x] Writer lässt sich über das Startmenü (K-Symbol links unten) starten. (Da hatten wir mal einen Bug).
* Datei-Assoziationen
    * [-] .md-Dateien werden mit Kate geöffnet.
		* nein, aktuell ist R-Studio eingestellt
    * [x] Firefox öffnet Ordner mit Dolphin (im Download-Dialog (nach Herunterladen einer Datei) -> Klick auf Ordner-Symbol)
* Tastenkombinationen
    * [-] WIN+T: öffnet ein Terminal
    * [-] WIN+K: öffnet KSnapshot (Programm für Bildschirmfotos)
    * [-] WIN+s: soll Gnome-orca öffnen
* FSFW-Doku
    * [x] In der Bookmark-Toolbar von Firefox ist ein Link auf unsere Dokumentation gut sichtbar
    * [x] Wenn man drauf klickt öffnet sich die Index-Seite mit dem FSFW-Layout
    * [ ] Alle Links funktionieren, insbesondere
        * [-] der Link auf unsere "Vollständige Paketliste", die bei github liegt (hier gab es einen Bug: #74)
			* aktuell wird auf eine alte version verlinkt, dies muss noch aktualisiert werden
        * [x] Links auf LaTeX-Vorlagen (Diplomarbeit und Präsentation öffnen korrekt in TeX-Studio und lassen sich kompilieren)
			* [ ] ERGÄNZUNG: Der Hinweis auf das angeblich aktuelle TexLive2016 sollte mal aktualisiert werden
    * [x] Index-Seite enthält die neu hinzugefügten FSFW-Links (Sprechstunde, Blog, ...)
		* ja, diese stehen in einem der letzten Absätze
    * [-] ERGÄNZUNG: Der Diaspora-Link sollte nicht ins Leere zeigen
    * [-] ERGÄNZUNG: Unsere Datenschutzrichtlinie sollte verlinkt sein
* Kate
    * [x] Zeigt beim ersten Start keinen Tipp-Dialog
    * [x] Verwendet zum Einrücken 4 Leerzeichen
    * [-] Entfernt beim Speichern überflüssige Leerzeichen am Zeilenende
* Hello-World-Collection
    * [x] Das Verzeichnis FSFW-Material/hello-world-collection/ existiert
    * [x] Es hat die Unterverzeichnisse C, go, Python
    * [x] Die C und go-Beispiele lassen sich kompilieren und ausführen
		* ERGÄNZUNG: Quelldateien sollten nicht automatisch mit R-Studio geöffnet werden (sofern es keine R-Dateien sind)
    * [-] Die Python-Beispiele lassen sich ohne Fehlermeldungen ausführen (entsprechend der Anweisungen in python/README.md)
		* Beispiel 3 funktioniert nicht, da python3-flask nicht installiert
		* [ ] Beispiel 5 testen, ggf. Anleitung anpassen
	* ERGÄNZUNG: Ein R-Beispiel wäre gut

**Comment by cknoll** [comment]
@Nos- Danke fürs Testen.

Ich denke es sind zu viele Dinge, um sie jetzt noch zu fixen. Wir müssen priorisieren.

Nicht so wichtig sind jetzt aus meiner Sicht:

Tastenkombinationen
Datenschutzrichtlinie
Accessibility (-> Bei Bedarf separaten Support anbieten)
Zotero (Literaturverwaltung nicht so wichtig für Erstis)
R-Beispiel: wenn wir jetzt keins haben, gibt es halt erstmal keins.


Flask-Beispiel kann gestrichen werden -> ich kümmere mich
Diaspora-Link im Template streichen: ich kümmere mich
gesperrter Bildschirm: ich bin für leeres PW oder Abschalten des Sperrens

**Comment by Nos-** [comment]
So, habe jetzt nochmal die ein oder andere Seite unserer Doku korrekturgelesen und Tippfehler korrigiert bzw. Angaben aktualisiert.
Lediglich für Links aufs Github-Repo (zumeist vom Repo zum Repo) müssten ggf. nochmal angepasst werden wegen Branches. Ich bin dafür auf den Masterbranch zu verlinken, da ja eh dorthin gemerged werden soll.

**Comment by Nos-** [comment]
Leeres PW geht durch Rauslöschen aus der shadow.

**Comment by Nos-** [comment]
> Habe ein paar Sachen gefixt.
> 
> @eMPee584 Was noch so mittel-wichtig wäre:
> 
>     * [ ]   md-Assoziation mit kate statt mit R-Studio
> 
>     * [ ]   Mauszeiger wieder klein, weil sonst z.B. der Text-Select-Curso groß und unpräzise ist

Ja, sehe ich auch so. Allerdings war der Mauszeiger erst seit dem Build von heute Nachmittag so rießig. Gestern war er (zumindest bei mir) noch normalgroß.
